Tests/PreviewSnapshotsTests/PreviewSnapshotsTests.swift

```diff
@@ -40,33 +40,7 @@ final class PreviewSnapshotsTests: XCTestCase {
             }
         )
         
-        snapshots.assertSnapshots(as: .testStrategy)
-    }
-    
-    /// PreviewSnapshots assertion using `named` parameter
-    func test_namedAssertion() {
```

Comment:
It was just added in the last PR, but this test felt redundant now that every test is using the `named` argument